spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+    factory :user do
+      email { 'test@gmail.com' }
```

Comment:
At some point, we could have a constraint where user's email cannot be duplicated, `sequence` might become handy

```ruby
factory :user do
  sequence(:email) { |n| "user#{n}@email.com" }
  password { 'thisistestpassword' }
end
```